sdks/python/apache_beam/yaml/examples/resources/products.csv

```diff
@@ -0,0 +1,6 @@
+transaction_id,product_name,category,price
+T0012,Headphones,Electronics,59.99
+T5034,Leather Jacket,Apparel,109.99
+T0024,Aluminum Mug,Kitchen,29.99
+T0104,Headphones,Electronics,59.99
+T0302,Monitor,Electronics,249.99
```

Comment:
do we need to this csv file? can we just create a temp csv for your example?

Reply:
I think that would defeat the purpose of the example - the point is for it to be runnable as is

Reply:
Fixed by moving csv to `apache-beam-testing`.